pkg/util/maven/maven_command.go

```diff
@@ -240,6 +245,11 @@ func generateProjectStructure(context Context, project Project) error {
 	return nil
 }
 
+// Create a MAVEN_CONTEXT file containing all arguments for a maven command
+func generateMavenContext(context Context, args []string) error {
+	return util.WriteToFile(filepath.Join(context.Path, "MAVEN_CONTEXT"), strings.Join(args, " "))
```

Comment:
What is this doing?

Reply:
In short : I write into a MAVEN_CONTEXT file all the parameters of the build maven command, and then I re-use them in the jib task. The way I see it is that we are doing maven command in both case, so any maven repository, or any other configuration used in the build context should work in the jib task.

But that's clearly the part that needs to be refined. I suspect I write too many arguments in the file. To illustrate this for better understanding that is why I need to remove the goal in the jib task.